app/controllers/hub/clients_controller.rb

```diff
@@ -133,8 +132,16 @@ def update_take_action
 
     private
 
+    def load_vita_partners
+      @vita_partners = VitaPartner.accessible_by(Ability.new(current_user))
+    end
+
     def form_params
-      params.require(:hub_client_intake_form).permit(ClientIntakeForm.permitted_params)
+      params.require(ClientIntakeForm.form_param).permit(ClientIntakeForm.permitted_params)
```

Comment:
change ClientIntakeForm.form_param to @form.class.form_params for ease of changing form name

Reply:
boo this didn't work because we havent instantiated the @form variable until we need to use the params. I do think it makes sense to refactor eventually, but it would take a lot more finagling than i think makes sense to do at the moment.